docs/user-guide/workflow.ipynb

```diff
@@ -86,7 +86,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "workflow = pol.he3.He3CellWorkflow(in_situ=False, incoming_polarized=True)\n",
+    "workflow = pol.he3.He3CellWorkflow(in_situ=False, incoming_polarized=False)\n",
```

Comment:
Should not be changed to `False`, since that is not what the text above says.

Reply:
Yes thats right, I forgot to switch it back after trying several things out.